### PR TITLE
Refactor test cases

### PR DESCRIPTION
**Pull request by Thing1Thing2**
Modify test cases to change "person" to "client" and "addressBook" to "clientBook".

This PR has many refactorings which may cause confusion with how it maps to the original repo.
Please comment to clarify any issues.

### CODE REVIEW

**Comment by Thing1Thing2** [comment]
Also, all references to "addressBook" should be changed to "myInsuRec"

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -33,15 +33,15 @@ public DeleteCommand(Index targetIndex) {
...
         if (targetIndex.getZeroBased() >= lastShownList.size()) {
-            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+            throw new CommandException(Messages.MESSAGE_INVALID_CLIENT_DISPLAYED_INDEX);
         }
 
-        Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
-        model.deletePerson(personToDelete);
-        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, personToDelete));
+        Client personToDelete = lastShownList.get(targetIndex.getZeroBased());
+        model.deleteClient(personToDelete);
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:43]: model.deleteClient(clientToDelete)

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -34,14 +34,14 @@ public class AddCommand extends Command {
             + PREFIX_TAG + "owesMoney";
 
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_CLIENT = "This person already exists in the address book";
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:37]: "This client already exists in the address book"

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 3]
```diff
@@ -34,14 +34,14 @@ public class AddCommand extends Command {
...
 
-    private final Person toAdd;
+    private final Client toAdd;
 
     /**
      * Creates an AddCommand to add the specified {@code Person}
      */
-    public AddCommand(Person person) {
+    public AddCommand(Client person) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:44]: Client client

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 4]
```diff
@@ -22,7 +22,7 @@ public class DeleteCommand extends Command {
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_CLIENT_SUCCESS = "Deleted Person: %1$s";
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:25]: Deleted Client

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 5]
```diff
@@ -46,60 +46,60 @@ public class EditCommand extends Command {
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
+    public static final String MESSAGE_EDIT_CLIENT_SUCCESS = "Edited Person: %1$s";
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:49]: Edited CLient

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 6]
```diff
@@ -46,60 +46,60 @@ public class EditCommand extends Command {
...
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_CLIENT = "This person already exists in the address book.";
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:51]: "This client already exists in the address book."

File: src/main/java/seedu/address/model/client/Client.java [hunk 7]
```diff
@@ -83,11 +83,11 @@ public boolean equals(Object other) {
             return true;
         }
 
-        if (!(other instanceof Person)) {
+        if (!(other instanceof Client)) {
             return false;
         }
 
-        Person otherPerson = (Person) other;
+        Client otherPerson = (Client) other;
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/model/client/Client.java:90]: change `otherPerson` to `otherClient`

File: src/main/java/seedu/address/model/client/NameContainsKeywordsPredicate.java [hunk 8]
```diff
@@ -8,15 +8,15 @@
...
+public class NameContainsKeywordsPredicate implements Predicate<Client> {
     private final List<String> keywords;
 
     public NameContainsKeywordsPredicate(List<String> keywords) {
         this.keywords = keywords;
     }
 
     @Override
-    public boolean test(Person person) {
+    public boolean test(Client person) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/model/client/NameContainsKeywordsPredicate.java:19]: same issue

File: src/main/java/seedu/address/model/client/UniqueClientList.java [hunk 9]
```diff
@@ -53,45 +53,46 @@ public void add(Person toAdd) {
      * {@code target} must exist in the list.
      * The person identity of {@code editedPerson} must not be the same as another existing person in the list.
      */
-    public void setPerson(Person target, Person editedPerson) {
+    public void setClient(Client target, Client editedPerson) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/model/client/UniqueClientList.java:56]: same issue

File: src/main/java/seedu/address/model/client/UniqueClientList.java [hunk 10]
```diff
@@ -53,45 +53,46 @@ public void add(Person toAdd) {
...
-        requireNonNull(replacement);
-        internalList.setAll(replacement.internalList);
-    }
 
     /**
      * Replaces the contents of this list with {@code persons}.
      * {@code persons} must not contain duplicate persons.
      */
-    public void setPersons(List<Person> persons) {
+    public void setClients(List<Client> persons) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/model/client/UniqueClientList.java:92]: same issue

File: src/main/java/seedu/address/model/client/UniqueClientList.java [hunk 11]
```diff
@@ -124,10 +125,10 @@ public int hashCode() {
     /**
      * Returns true if {@code persons} contains only unique persons.
      */
-    private boolean personsAreUnique(List<Person> persons) {
+    private boolean clientsAreUnique(List<Client> persons) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/model/client/UniqueClientList.java:128]: same issue

File: src/main/java/seedu/address/model/client/exceptions/DuplicateClientException.java [hunk 12]
```diff
@@ -1,11 +1,11 @@
...
 
 /**
  * Signals that the operation will result in duplicate Persons (Persons are considered duplicates if they have the same
  * identity).
  */
-public class DuplicatePersonException extends RuntimeException {
-    public DuplicatePersonException() {
+public class DuplicateClientException extends RuntimeException {
+    public DuplicateClientException() {
         super("Operation would result in duplicate persons");
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/model/client/exceptions/DuplicateClientException.java:9]: same issue

File: src/main/java/seedu/address/storage/JsonAdaptedClient.java [hunk 13]
```diff
@@ -60,11 +60,11 @@ public JsonAdaptedPerson(Person source) {
...
     /**
-     * Converts this Jackson-friendly adapted person object into the model's {@code Person} object.
+     * Converts this Jackson-friendly adapted person object into the model's {@code Client} object.
      *
-     * @throws IllegalValueException if there were any data constraints violated in the adapted person.
+     * @throws IllegalValueException if there were any data constraints violated in the adapted client.
      */
-    public Person toModelType() throws IllegalValueException {
+    public Client toModelType() throws IllegalValueException {
         final List<Tag> personTags = new ArrayList<>();
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/storage/JsonAdaptedClient.java:68]: same issue

File: src/main/java/seedu/address/storage/JsonSerializableClientBook.java [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    /**
+     * Converts this client book into the model's {@code ClientBook} object.
+     *
+     * @throws IllegalValueException if there were any data constraints violated.
+     */
+    public ClientBook toModelType() throws IllegalValueException {
+        ClientBook clientBook = new ClientBook();
+        for (JsonAdaptedClient jsonAdaptedPerson : clients) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/storage/JsonSerializableClientBook.java:50]: same issue

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 15]
```diff
@@ -110,13 +110,13 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
-        personListPanel = new PersonListPanel(logic.getFilteredPersonList());
+        personListPanel = new PersonListPanel(logic.getFilteredClientList());
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/ui/MainWindow.java:113]: same issue

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 16]
```diff
@@ -44,7 +44,7 @@ public class PersonCard extends UiPart<Region> {
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
      */
-    public PersonCard(Person person, int displayedIndex) {
+    public PersonCard(Client person, int displayedIndex) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/ui/PersonCard.java:47]: same issue

File: src/main/java/seedu/address/ui/PersonListPanel.java [hunk 17]
```diff
@@ -18,12 +18,12 @@ public class PersonListPanel extends UiPart<Region> {
...
 
     @FXML
-    private ListView<Person> personListView;
+    private ListView<Client> personListView;
 
     /**
      * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
      */
-    public PersonListPanel(ObservableList<Person> personList) {
+    public PersonListPanel(ObservableList<Client> personList) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/ui/PersonListPanel.java:26]: same issue

File: src/main/java/seedu/address/ui/PersonListPanel.java [hunk 18]
```diff
@@ -32,9 +32,9 @@ public PersonListPanel(ObservableList<Person> personList) {
     /**
      * Custom {@code ListCell} that displays the graphics of a {@code Person} using a {@code PersonCard}.
      */
-    class PersonListViewCell extends ListCell<Person> {
+    class PersonListViewCell extends ListCell<Client> {
         @Override
-        protected void updateItem(Person person, boolean empty) {
+        protected void updateItem(Client person, boolean empty) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/ui/PersonListPanel.java:37]: same issue

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 19]
```diff
@@ -27,21 +27,21 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
 import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
 import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
-import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_CLIENT;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_CLIENT;
 import static seedu.address.testutil.TypicalIndexes.INDEX_THIRD_PERSON;
```
Comment (ThomasHoooo) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:32]: same issue

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 20]
```diff
@@ -203,7 +203,7 @@ public void parse_resetTags_success() {
         Index targetIndex = INDEX_THIRD_PERSON;
```
Comment (ThomasHoooo) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:203]: same issue

File: src/test/java/seedu/address/storage/JsonAdaptedClientTest.java [hunk 21]
```diff
@@ -33,78 +33,78 @@ public class JsonAdaptedPersonTest {
             .collect(Collectors.toList());
 
     @Test
-    public void toModelType_validPersonDetails_returnsPerson() throws Exception {
-        JsonAdaptedPerson person = new JsonAdaptedPerson(BENSON);
+    public void toModelType_validClientDetails_returnsPerson() throws Exception {
+        JsonAdaptedClient person = new JsonAdaptedClient(BENSON);
```
Comment (ThomasHoooo) [inline src/test/java/seedu/address/storage/JsonAdaptedClientTest.java:37]: same issue

**Review by ThomasHoooo — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 1 above)
Comment (Thing1Thing2) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: As with the follow up requests, references to person have been changed.

**Review by ThomasHoooo — APPROVED** [review]
LGTM

**Pull request merged**